packages/tests/unit-tests/helpers/webpack-test-compiler.ts

```diff
@@ -39,10 +39,11 @@ export default (
   return new Promise<webpack.Stats>((resolve, reject) => {
     compiler.run((err, stats) => {
       if (err) reject(err);
-      if (stats.hasErrors())
-        reject(new Error(stats.toJson().errors.join("\n")));
-
-      resolve(stats);
+      if (stats) {
```

Comment:
Can we use something like this?
if(stats && stats.hasErrors())

Reply:
Yes. I change to stats?.hasError()